src/electric/adapter.ts

```diff
@@ -1,19 +1,22 @@
 import { AnyDatabase } from '../drivers/index'
 import { QualifiedTablename } from '../util/tablename'
-import { BindParams, Query, Row } from '../util/types'
+import { Row, Statement } from '../util/types'
 
 // A `DatabaseAdapter` adapts a database client to provide the
 // normalised interface defined here.
 export interface DatabaseAdapter {
   db: AnyDatabase
 
-  // Runs the provided sql.
-  run(sql: string): Promise<void>
+  // Runs the provided sql statement
+  run(statement: Statement): Promise<void>
```

Comment:
Is there a use case for `run`? Is it just that a single statement doesn't need a transaction? In which case, makes sense.

Reply:
Single statement doesn't need a transaction.